ci/test.yml

```diff
@@ -0,0 +1,17 @@
+image: $CI_DEPENDENCY_PROXY_GROUP_IMAGE_PREFIX/nixos/nix:latest
+
+test:
+  stage: test
```

Comment:
Connected to the first comment I would run this In the stage after we build the image and test the image that was built.

Reply:
(Together with that first comment of yours):

That is right that there might be slightly different behavior in production image - so in `master` branch, we should really test the whole `tenv:latest` image after being built.

However, for normal branches, I would leave only these quicker tests in that `basic nixos docker` image, as that does not need any building from our side (and is therefore much quicker and less resource-demanding)

Reply:
Agree. Let's do it this way then.